app/move/index.js

```diff
@@ -24,6 +33,13 @@ const createConfig = {
   journeyName: 'create-a-move',
   journeyPageTitle: 'actions::create_move',
 }
+const updateConfig = {
+  ...wizardConfig,
+  controller: update.Base,
+  templatePath: 'move/views/create/',
+  template: '../../../form-wizard',
+  journeyPageTitle: 'actions::update_move',
```

Comment:
Maybe also add a different `journeyName`. That way it will prevent any clashes on the session namespace.

Reply:
Ah, it's down down below. Ignore this comment then.

Reply:
As noted, covered below